packages/lesswrong/components/common/Home2.jsx

```diff
@@ -37,22 +59,40 @@ class Home2 extends PureComponent {
   
         <SingleColumnSection>
           <SectionTitle title="Recent Discussion">
-            {shortformFeedId &&  <div onClick={this.toggleShortformFeed}>
+            {/* {shortformFeedId &&  <div onClick={this.toggleShortformFeed}>
               <SectionButton>
                 <AddBoxIcon />
                 New Shortform Post
               </SectionButton>
-            </div>}
+            </div>} */}
           </SectionTitle>
-          {showShortformFeed && <CommentsNewForm
+          <SectionFooter>
+            <Tooltip title={"Toggle whether recent discussion comments are expanded"}>
```

Comment:
Does this really need to be on the front page? It seems like putting it on the edit profile page would be about as good, and use up less of the UI-complexity budget.

Reply:
It seems pretty important for people to be able to quickly expand all and un-expand all without having to navigate over to their user page. There's a few different ways to go about it (maybe hidden beneath a gear icon) but I think it at least needs to be on frontpage.